src/pages/Defi/components/OpportunityCard.tsx

```diff
@@ -13,7 +13,9 @@ import {
 import { cosmosChainId } from '@shapeshiftoss/caip'
 import { DefiType } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
 import { EarnOpportunityType } from 'features/defi/helpers/normalizeOpportunity'
+import { FoxAssetId, FoxyIconUrl } from 'plugins/foxPage/foxPage'
```

Comment:
please don't import these from this plugin folder - plugins are supposed to be used in isolation, and data from them shouldn't be consumed outside of them